static/jquery.js

```diff
@@ -0,0 +1,5 @@
+/*! jQuery v2.1.4 | (c) 2005, 2015 jQuery Foundation, Inc. | jquery.org/license */
```

Comment:
Why is this file needed?

Reply:
jQuery is a third party JavaScrip library. This is just a backup of the jQuery library. The client needs to get this library in order to work. Now, if the client can't get it for some reason, we can give it to them ourselves. You can spin up the server, turn off wifi, and comment out the following code to see it break. Then, add the following code back in to see it work.
```
    <script>
      window.jQuery || document.write(
        '<script src="{{ url_for('static', filename='jquery.js') }}">\x3C/script>'
      )
    </script>
```
All this code is is an `or` statement. "Does the window already have `window.jQuery`? If not, get it from `static/jquery.js`."

Reply:
I see. Is it common to do this?

Reply:
Very common. It's my bad that I didn't do it in the first place. Imagine if clients had to pull all of the dependancies from pipy every time. It would be pretty bad if pipy went down.